src/main/java/org/opensearch/security/auditlog/config/AuditConfig.java

```diff
@@ -400,58 +444,65 @@ public void log(Logger logger) {
 
         @Override
         public String toString() {
-            return "Filter{" +
-                    "isRestApiAuditEnabled=" + isRestApiAuditEnabled +
-                    ", disabledRestCategories=" + disabledRestCategories +
-                    ", isTransportApiAuditEnabled=" + isTransportApiAuditEnabled +
-                    ", disabledTransportCategories=" + disabledTransportCategories +
-                    ", resolveBulkRequests=" + resolveBulkRequests +
-                    ", logRequestBody=" + logRequestBody +
-                    ", resolveIndices=" + resolveIndices +
-                    ", excludeSensitiveHeaders=" + excludeSensitiveHeaders +
-                    ", ignoredAuditUsers=" + ignoredAuditUsersMatcher +
-                    ", ignoreAuditRequests=" + ignoredAuditRequestsMatcher +
-                    '}';
+            return "Filter{"
```

Comment:
(Comment on the code) This is a good case for a string builder with format appends for variable pairs. This is terrible performance and heap utilization to allocation each temp substring.

Reply:
Should we create an issue here?

Reply:
I don't think so. While there are significant drawbacks to its implementation - it's working code. 

Worth revisiting if they were reports of customer impact